gradio/components.py

```diff
@@ -1506,6 +1532,15 @@ def stream(
             _postprocess=_postprocess,
         )
 
+    def as_example(self, input_data):
```

Comment:
We can add this validation to other components to improve the developer experience, e.g. Audio, Video, CheckboxGroup. We can also get really detailed if we want, e.g. Image files must end in png or jpg